src/NZGBplugin/Ui_NewFeatureDialog.py

```diff
@@ -1,101 +1,75 @@
-# -*- coding: utf-8 -*-
-################################################################################
-#
-#  New Zealand Geographic Board gazetteer application,
-#  Crown copyright (c) 2020, Land Information New Zealand on behalf of
-#  the New Zealand Government.
-#
-#  This file is released under the MIT licence. See the LICENCE file found
-#  in the top-level directory of this distribution for more information.
-#
-################################################################################
```

Comment:
Where's ther copyright header gone ?

Reply:
These files are generated via pyuic5 (from .ui > py files) and headers will fall out each time they are run through this utility. For now I have added it back in manually.

Reply:
Does it make sense to include generated files in a git repository ? I would not do that, delegating it to packaging